plugins/sensors/upsmonpro_

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+upsmonpro_ - Munin plugin to monitor Powercom UPS via UPSMON PRO program L<http://www.pcm.ru/support/soft/>
+
+=head1 INSTALLATION
+
+  /etc/munin/plugins/upsmonpro_load -> /usr/share/munin/plugins/upsmonpro_
+  /etc/munin/plugins/upsmonpro_status -> /usr/share/munin/plugins/upsmonpro_
+  /etc/munin/plugins/upsmonpro_temp -> /usr/share/munin/plugins/upsmonpro_
+  /etc/munin/plugins/upsmonpro_voltage -> /usr/share/munin/plugins/upsmonpro_
+
+=head1 CONFIGURATION
+
+Environment variables:
+
+  host - UPSMON PRO server host, default localhost
+  port - UPSMON PRO port, default 2601
+
+=head1 MAGIC MARKERS
+
+  #%# family=auto
+  #%# capabilities=autoconf
+
+=head1 AUTHOR
+
+Copyright (C) 2017 pru.mike@gmail.com
+
+=head1 LICENSE
+
+GPLv2.
+
+=cut
+
+use strict;
+use warnings;
+use feature qw/say/;
+use Munin::Plugin;
+use IO::Socket::INET;
+use Time::HiRes qw/usleep/;
+use English qw/-no-math-vars/;
+our $VERSION = '0.0.1';
+$OUTPUT_AUTOFLUSH++;
+
+our $DEFAULT_HOST = 'localhost';
+our $DEFAULT_PORT = 2601;
+our %TYPES        = (
+  voltage => [qw/input output/],
+  load    => [qw/battery_load battery_capacity/],
+  temp    => [q/temp/],
+  status  => [qw/power_failure low_battery voltage_status ups_status  battery_test/]
+);
+our @TYPES = keys %TYPES;
+
+{
+  no strict 'refs';
+  for my $t (@TYPES) {
+    for my $sub (qw/autoconf/) {
+      *{ __PACKAGE__ . "::run_${sub}_$t" } = "run_$sub";
```

Comment:
My perl capabalities are quite rusty, thus I cannot judge, if you are using a common approach for dynamic function calls here.
If you think, this is easily understandable, then leave it like it is.
Otherwise I would suggest to use something less concise, but easier to read in order to make life easier for future contributors.